src/Builder/FormContractor.php

```diff
@@ -154,7 +152,7 @@ public function getDefaultOptions($type, FieldDescriptionInterface $fieldDescrip
                     .' Maybe you want `%s` instead?',
                     AdminType::class,
                     $fieldDescription->getName(),
-                    CollectionType::class
+                    'Sonata\Form\Type\CollectionType'
```

Comment:
Should we add a Todo here (and for every others changes), to use `::class` again when possible ?

Reply:
I've omitted these comments because I thought it will be an obvious decision, but I can add them if there is consensus about the requirement.

Reply:
IMO, adding a comment
- Will help to not forget,
- Find all changes to do in one grep
- Will help to know the new value 

For example, what will be the new value when I see `'Sonata\Form\Type\CollectionType'` and `'Sonata\CoreBundle\Form\Type\CollectionType'`, I can't be sure what to do, since there is only one CollectionType in the form-extension.

Reply:
Do you think you could be in doubt about the decision between a class that belongs to `Sonata\Form\` namespace and other that belongs to `Sonata\CoreBundle\`? :thinking: 
The user already knows that SonataCoreBundle is deprecated. Even, there will be no choice that allows "sonata-project/core-bundle" to be installed in the same project.
Anyway, I've added the comments.